phpstan.neon.dist

```diff
@@ -23,3 +23,41 @@ parameters:
 		- bin/phpstan/filtered-functions.stub
 	dynamicConstantNames:
 		- PERFLAB_OBJECT_CACHE_DROPIN_VERSION
+	ignoreErrors:
+		-
+			# Ignore error related to PHP 8 using a GdImage but PHP 7 using a resource.
```

Comment:
indentation of this comment line doesn't match those a few lines below

Reply:
Hummm. I guess because I was commenting on this one single error, whereas the comment below is for all of the remaining errors. If that's not clear, we can move the comment up to the line before `#`.

Reply:
no worries, just looks weird visually.